packages/editor/src/components/post-saved-state/index.js

```diff
@@ -100,7 +98,6 @@ export class PostSavedState extends Component {
 		if ( forceSavedMessage || ( ! isNew && ! isDirty ) ) {
 			return (
 				<span className="editor-post-saved-state is-saved">
-					<Icon icon={ check } />
 					{ __( 'Saved' ) }
```

Comment:
I wonder if we can remove this "saved" state entirely (I mean even the text)

Reply:
I think we need to replace it with a snackbar notification (there currently isn't any), but yeah, I think we should get rid of the "Saved" state.

Reply:
You two partially crazy people... you might be on to something! I think it's perhaps best to try this separately — removing the saved state indicator, and replacing it with a snackbar. But it could work!